modules/tracking/samples/lasot_benchmark.py

```diff
@@ -0,0 +1,246 @@
+    via cv.TrackerX_create().
+    Input: string with tracker name.
+    Output: dictionary 'config'
+    Dictionary 'config' contains trackers names
+    as keys and tuple with call method and number of frames for
+    reinitialization as values
+    '''
+    config = {"Boosting": (cv.TrackerBoosting_create(), 500),
+              "MIL": (cv.TrackerMIL_create(), 1000),
+              "KCF": (cv.TrackerKCF_create(), 1000),
+              "MedianFlow": (cv.TrackerMedianFlow_create(), 1000),
+              "GOTURN": (cv.TrackerGOTURN_create(), 250),
+              "MOSSE": (cv.TrackerMOSSE_create(), 1000),
+              "CSRT": (cv.TrackerCSRT_create(), 1000)}
+    return config[tracker_name]
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        description="Run LaSOT-based benchmark for visual object trackers")
+    # As a default argument used name of
+    # original dataset folder
+    parser.add_argument("--dataset", type=str,
+                        default="LaSOTTesting", help="Full path to LaSOT")
+    parser.add_argument("--v", dest="visualization", action='store_true',
+                        help="Showing process of tracking")
+    args = parser.parse_args()
+
+    # Creating list with names of videos via reading names from txt file
+    video_names = os.path.join(args.dataset, "testing_set.txt")
+    with open(video_names, 'rt') as f:
+        list_of_videos = f.read().rstrip('\n').split('\n')
+    trackers = [
+        'Boosting', 'MIL', 'KCF', 'MedianFlow', 'GOTURN', 'MOSSE', 'CSRT']
+
+    list_iou = []
+    list_pr = []
+    list_n_pr = []
+
+    # Loop for every tracker
+    for tracker_name in trackers:
+
+        print("Tracker name: ", tracker_name)
+
+        number_of_thresholds = 21
+        iou_video = np.zeros(number_of_thresholds)
+        pr_video = np.zeros(number_of_thresholds)
+        n_pr_video = np.zeros(number_of_thresholds)
+        iou_thr = np.linspace(0, 1, number_of_thresholds)
+        pr_thr = np.linspace(0, 50, number_of_thresholds)
+        n_pr_thr = np.linspace(0, 0.5, number_of_thresholds)
+
+        # Loop for every video
+        for video_name in list_of_videos:
+
+            tracker, frames_for_reinit = init_tracker(tracker_name)
+            init_once = False
+
+            print("\tVideo name: " + str(video_name))
+
+            # Open specific video and read ground truth for it
+            gt_file = open(os.path.join(args.dataset, video_name,
+                           "groundtruth.txt"), "r")
+            gt_bb = gt_file.readline().rstrip("\n").split(",")
+            init_bb = tuple([float(b) for b in gt_bb])
+
+            # Creating blob from image sequence
+            video_sequence = sorted(os.listdir(os.path.join(
+                args.dataset, video_name, "img")))
+
+            # Variables for saving sum of every metric for every frame and
+            # every video respectively
+            iou_list = []
+            pr_list = []
+            n_pr_list = []
```

Comment:
We need to find better name for these variables. Above you have 
```python
    list_iou = []
    list_pr = []
    list_n_pr = []
```
If you give a suitable name for variables, comments are not needed

Reply:
OK. I will change some names of the variables.